MonitorizareVot/Base.lproj/Localizable.strings

```diff
@@ -33,6 +33,9 @@
 "Label_SelectOption" = "Select an option";
 "Label_DevelopedBy" = "developed by";
 
+"Label_TelephoneTextInput_Placeholder" = "Phone number";
+"Label_CodeTextInput_Placeholder" = "Code";
```

Comment:
Not sure what the plan is in grouping localisation keys, so happy to update if it doesn't fit into that plan.

Reply:
Alex, right now it's grouped as good as it could be after I cleaned up all the extra stuff. If you want to group them better, go ahead. But please create and issue first then work on it so we can reference it. Thanks for your work so far!